packages/expo-task-manager/android/src/main/AndroidManifest.xml

```diff
@@ -1,9 +1,10 @@
-<manifest package="expo.modules.taskManager"
-          xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="expo.modules.taskManager">
+
     <application>
         <receiver
             android:name=".TaskBroadcastReceiver"
-            android:exported="false">
+            android:exported="true">
```

Comment:
Is this intended?

Reply:
Yes and no. It was intended for testing, but should (and will) be reverted.